torch/ao/quantization/pt2e/prepare.py

```diff
@@ -62,6 +64,9 @@ def _maybe_insert_input_observer_for_arg_or_kwarg(
     arg_as_output_act_obs_or_fq = _get_output_act_obs_or_fq(arg, named_modules, obs_or_fq_map, is_qat)
     arg_as_output_target_dtype, arg_as_output_target_is_dynamic = _get_dtype_and_is_dynamic(arg_as_output_act_obs_or_fq)
 
+    input_qspec_map = quantization_annotation.input_qspec_map
+    input_arg_qspec = _get_qspec_for_arg(arg, input_qspec_map, named_modules)
```

Comment:
This is used only when you replacing already existing observer code. Maybe move it closer to the use?